README.rst

```diff
@@ -30,14 +29,14 @@ This benchmark can be run using the following commands:
    $ git clone https://github.com/benchopt/benchmark_mcp
    $ benchopt run ./benchmark_mcp
 
-Apart from the problem, options can be passed to `benchopt run`, to restrict the benchmarks to some solvers or datasets, e.g.:
+Apart from the problem, options can be passed to ``benchopt run``, to restrict the benchmarks to some solvers or datasets, e.g.:
```

Comment:
Excuse these - out of the scope of the PR  - changes.

It's either now or never